index.html

```diff
@@ -13,6 +13,7 @@
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
+        <i class="fas fa-times"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
```

Comment:
when I click on a mobile menu option, the mobile menu should disappear as stated in the project requirements, here it's not working, kindly fix this issue.